Expensify/App
# fix "Chat - Keyboard is displayed on header menu screen after sending a file larger than 24 MB"

### Details
Straight revert of #30864

### Fixed Issues
$ https://github.com/Expensify/App/issues/31426


### Tests
Same as QA


### Offline tests
Same as QA

### QA Steps
1. Go to any chat and focus on composer
2. Try to send a file larger than the 24 MB limit
3. Close the "Attachment too large" modal
4. Tap on the chat header
5. Verify that keyboard doesn't show on newly opened page


### PR Author Checklist

### Screenshots/Videos
<details>
<summary>Android: Native</summary>


https://github.com/Expensify/App/assets/108292595/5ae80988-d59d-4575-905d-a74492b64d09



</details>

<details>
<summary>Android: mWeb Chrome</summary>


https://github.com/Expensify/App/assets/108292595/dfddc0e8-dd38-4de8-b17a-ca7589a40ca7



</details>

<details>
<summary>iOS: Native</summary>


https://github.com/Expensify/App/assets/108292595/68e39081-2cdc-4b0f-91f2-bd1f3c40f1a5



</details>

<details>
<summary>iOS: mWeb Safari</summary>


https://github.com/Expensify/App/assets/108292595/cb0c6385-e435-4543-bc45-80e040e11da0



</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



</details>

<details>
<summary>MacOS: Desktop</summary>



</details>
